### PR TITLE
Add the github actions for gh-pages deployment

### PR DESCRIPTION
**Pull request by huningxin**
This PR will simplify the deployment workflow.

@BruceDai @anssiko , PTAL. Thanks!

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>WebNN Code Editor</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
-    <link rel="stylesheet" href="./node_modules/codemirror/lib/codemirror.css">
-    <link rel="stylesheet" href="./node_modules/codemirror/theme/railscasts.css">
+    <link rel="stylesheet" href="./libs/codemirror/codemirror.css">
```
Comment (BruceDai) [inline code/index.html:7]: @huningxin Now it doesn't depend on the **codemirror** node module, so please remove package.json and update the README.md, thanks.

File: lenet/README.md [hunk 2]
```diff
@@ -83,8 +83,6 @@ Install dependencies:
 > npm install
```
Comment (BruceDai) [inline lenet/README.md:83]: @huningxin Now it doesn't depend on the **mnist** node module, so please remove package.json and update the README.md, thanks.

**Review by BruceDai — COMMENTED** [review]
@huningxin I left some comments for code/lenet samples, please update it , thanks.

File: code/index.html (hunk 1 above)
Comment (huningxin) [reply code/index.html:7]: Good catch. Added a new commit to solve this issue. Please take another look.

File: lenet/README.md (hunk 2 above)
Comment (huningxin) [reply lenet/README.md]: ditto

**Comment by BruceDai** [comment]
LGTM, thanks @huningxin

**Review by BruceDai — APPROVED** [review]

**Review by anssiko — APPROVED** [review]

**Comment by huningxin** [comment]
Thanks for the review and approvals.

**Pull request merged**